### PR TITLE
Link character carousel to character profile pages

### PR DESCRIPTION
**Pull request by chenga3**
Includes changes from #257 <- probs merge this first before reviewing this.

Buttons on carousel page now link to the character's profile page. The 'other characters' button on the profile page also links back to the carousel page. Also did some minor reorganising.

### CODE REVIEW

**Comment by CodeWithTheDoctor** [comment]
`sw.js.*` files should be address to gitignore i think. Don't include them
in the commit

**Comment by chenga3** [comment]
> `sw.js.*` files should be address to gitignore i think. Don't include them in the commit
> […](#)

https://github.com/vercel/next.js/tree/canary/examples/progressive-web-app/public
Im guessing also the workbox files currently in the repo? They aren't included here in this example PWA ^

File: pages/profile/[profile].tsx [hunk 1]
```diff
@@ -1,52 +1,49 @@
...
+import bgImg from 'public/images/BG Blue.jpg'
+
+interface Props {
+  /** All characters from the town specified in the URL. Fetched from Notion DB. */
+  character: Character
+}
+
+/** Page displaying carousel of characters from a particular town. Contains links to individual character pages. */
+const Profile = ({ character }: Props) => {
+  const [state, setState] = useState(true)
```
Comment (CodeWithTheDoctor) [inline pages/profile/[profile].tsx:18]: Please change the name of this state to something with a more semantic meaning such that other developers will understand what its purpose could be.

**Review by CodeWithTheDoctor — APPROVED** [review]
Looks good! Great work!

**Pull request merged**